app/assets/javascripts/components/articles/article.jsx

```diff
@@ -14,7 +14,7 @@ const Article = ({ article, index, course, fetchArticleDetails, updateArticleTra
 
   const fetchMissingArticleDetails = () => {
     if (!articleDetails) {
-      fetchArticleDetails(article.id, course.id);
+      fetchArticleDetails(article.id, course.id, article.title, article.mw_page_id);
```

Comment:
I think this gets called from several places, including Assignment.jsx which you haven't changed. Did you check them all to verify that this doesn't break anything?

Reply:
> Did you check everything to ensure this doesn't break anything?

No 😅, I forgot to check.
Now, I have updated fetchArticleDetails to fetch the article title and the MW page ID directly from the Redux store using the existing article ID via getState. This means other places using fetchArticleDetails no longer need to pass the title and page ID.